src/services/dex.js

```diff
@@ -1142,11 +1179,24 @@ export default {
     });
   },
 
-  fetchTradesBucketed(marketName, binSize = 1) {
+  fetchTradesBucketed(marketName, binSize = 1, from = null, to = null) {
```

Comment:
No need for these default parameter values as they will resolve to `undefined` which is `falsey`.

Reply:
merged this from another's change from desktop. I'll remove the default params.